### PR TITLE
Electronic documents, chapter shortname, pa11y-ci update

### PR DESCRIPTION
**Pull request by dmundra**
A lot of changes, here is a summary:

1. Renaming 'Electronic Docs' to 'Electronic Documents' for #280 
2. Added `short_label` to the chapter object for the catalog schema so we don't have to have a separate file in the editor for those values.
3. Updated pa11y-ci to latest version so it running more up to date axe-core checks. Did have to ignore contract checks because it cannot calculate the contrast of the table header.
4. With the package changes I updated the license file. See https://github.com/GSA/openacr/tree/main/license for details.

**QA**

1. Review the HTML and Markdown files to confirm the rename of 'Electronic Docs' to 'Electronic Documents'. You can download generated HTML from the GitHub action: https://github.com/GSA/openacr/actions/runs/1725653905
2. Short label will be tested in the editor.
3. Confirm the a11y-tests run without errors.

### CODE REVIEW

File: catalog/2.4-edition-wcag-2.0-508-en.yaml [hunk 1]
```diff
@@ -572,7 +576,7 @@ chapters:
           - none
   - id: hardware
     label: "Chapter 4: Hardware"
-    chapter_title: Chapter 4
```
Comment (dmundra) [inline catalog/2.4-edition-wcag-2.0-508-en.yaml:575]: Found a random lingering key value pair that we didn't end up using.

**Comment by dmundra** [comment]
@qjac I tagged @mgifford as a reviewer as well just so he is aware of changes but I want you to be the primary so you can review the code and ask questions. Thank you!

**Comment by qjac** [comment]
"Short label will be tested in the editor." @dmundra 

Is there a url to test or do I pull it down locally to QA?

**Comment by dmundra** [comment]
> "Short label will be tested in the editor." @dmundra
> 
> Is there a url to test or do I pull it down locally to QA?

Ah, the plan to add it here and then after this merges pull down the updated version in the editor and then use the new key there. So nothing to test here.

**Review by qjac — APPROVED** [review]
Looks good 👍

**Pull request merged**